internals/cli/cmd_run.go

```diff
@@ -195,14 +197,15 @@ func runDaemon(rcmd *cmdRun, ch chan os.Signal, ready chan<- func()) error {
 	}
 
 	dopts := daemon.Options{
-		Dir:        rcmd.pebbleDir,
-		SocketPath: rcmd.socketPath,
-		IDSigner:   idSigner,
+		Dir:          rcmd.pebbleDir,
+		SocketPath:   rcmd.socketPath,
+		IDSigner:     idSigner,
+		HTTPAddress:  rcmd.HTTP,
+		HTTPSAddress: rcmd.HTTPS,
```

Comment:
Do we need to create the `idSigner` if `HTTPAddress` is not specified? Seems a bit strange to be creating the "identity" directory above if it's never going to be used.

Reply:
Discussed with @benhoyt again. We are not used to extra stuff in $PEBBLE, but there is not really a consistent solution around this. Ben is OK for me to proceed.